wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -741,6 +746,10 @@ public final class SessionManager: NSObject, SessionManagerType {
             if deleteAccount {
                 self?.deleteAccountData(for: account)
             }
+
+            // Clear cache directory when the user logout from the session.
```

Comment:
This comment isn't really adding any new info that I can't already read from the context, I suggest to delete it.

Reply:
removed comment